lib/libzfs_core/libzfs_core.c

```diff
@@ -664,9 +717,18 @@ lzc_send_resume_redacted(const char *snapname, const char *from, int fd,
 {
 	nvlist_t *args;
 	int err;
+	int pipefd[2];
+	pthread_t mythread;
+	pthread_attr_t myattr;
+	sendargs_t sendargs;
+	int dumbstatus;
+
+
+	err = pipe(pipefd);
```

Comment:
Please create new fds with cloexec set (i.e. turn this into `pipe2(pipefd, O_CLOEXEC)`)

Reply:
Done!